GOVERNANCE.md

```diff
@@ -4,83 +4,138 @@ This document defines project governance for the Strimzi project.
 Voters should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
 
-### Approving PRs
+The project uses different kinds of vote for different purposes.
+
+* PR approval votes
+* Lazy majority votes
+* Explicit majority votes
 
-PRs may be merged after receiving at least two positive maintainer votes.
+Who may vote, where votes are recorded and the duration of the voting also depends on the subject of vote, as explained in the following sections.
+
+#### PR approval rules
+
+PRs against most repositories may be merged after receiving at least two positive maintainer votes.
 Voting for PRs can be done using a comment in the PR or by approving (or requesting changes) the PR.
 If the PR author is a maintainer, this counts as a vote.
 
-PRs which affect only a single component can be approved by at least one positive maintainer vote and one positive vote from an owner of given component.
+PRs which affect only a single component can be approved by at least one positive maintainer vote and one positive vote from an owner of the given component.
 If the PR author is a maintainer or an owner of given component, this counts as a vote.
 
-### Proposals
-
-Proposals can be opened against the [proposals repository](https://github.com/strimzi/proposals).
-Proposals should cover any changes to the Strimzi project which might significantly impact its users or the project direction.
+### Lazy majority rules
 
-Voting about proposals is using +1, 0, -1 votes and their fractions, where:
+Lazy majority votes use +1, 0, -1 votes and their fractions, where:
 * +1 means "yes"
 * -1 means "no"
 * The numbers between +1 and -1 indicate how strongly you feel about the proposal
 
 _(Inspired by [Apache Software Foundation voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions))_
 
-Votes from maintainers are binding and count towards the approval of the proposal.
-Non-maintainers are allowed and encouraged to vote as well.
-But their votes are non-binding and do not count towards approval of the proposal.
+Binding votes are those cast by people in specific roles defined for that vote and count towards the approval of the subject of the vote.
+Non-binding may be cast by people not in any of those roles and do not count towards approval of the subject of the vote.
 
-+1 votes can be expressed by approving the proposal PR or in the comments.
-Other votes should be expressed in the comments.
-For example "+1 (binding)", or for a non-maintainer "+1 (non-binding)".
+The vote succeeds when there are least three +1 binding votes and no -1 binding votes.
+The vote should be opened for at least 3 days to give everyone enough time to vote.
+
+### Explicit majority rules
+
+An explicit majority vote is simply a +1 or -1. 
+The vote succeeds when ⅔ of cast binding votes are +1.
+The vote should be opened for at least 7 days to give everyone enough time to vote.
```

Comment:
I'm fine with this change. But it is quite significant. So maybe you should better explain what does it change in the PR description to make sure it is obvious for everyone? Because it can get easily lost in the restructuring of the document.

Reply:
Done.